homework_2/turtle-stretch.js

```diff
@@ -0,0 +1,278 @@
+/* [Homework] Turtle Graphics - 2 */
```

Comment:
feel free to include stretch in non-stretch exercise so then you can only submit one file that has the logic for both